index.js

```diff
@@ -146,7 +146,7 @@ export const createImageProgress = ImageComponent =>
 
     onLoadEnd = event => {
       this.setState({
-        error: null,
+        ...this.state,
```

Comment:
This spread is redundant

Reply:
@oblador Fixed. I usually don't use local state in my projects so I'm not as familiar with it as I probably should be...